src/System.Reflection.TypeExtensions/ref/Configurations.props

```diff
@@ -2,9 +2,9 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp;
+      netstandard;
       netfx;
-      uap;
+      net461;
```

Comment:
Do all these need to be both build configurations or can some be moved to package configurations?